src/components/TableLine.tsx

```diff
@@ -3,7 +3,7 @@
  */
 
 import { FunctionComponent } from "react";
-import HorizontalLine from "./styled/HorizontalLine";
```

Comment:
I'd like to discuss this animation with the design team. I think we should only be adding animations where we are intentionally trying to draw the user's attention to that element.

Reply:
Ok I agree. Let's discuss with design and change if needed.

Reply:
Sounds good, thanks.

Reply:
For the time being I'll make it so slow that its barely noticable

Reply:
Ok